### ISSUE TITLE
Getting "license query fail" on Unity Asset Store version.

### ISSUE THREAD

**Issue by ghost** [Alteruna/au-multiplayer-wiki#3]
Downloaded through Unity Asset Store, imported to project, and the examples time out with a license query failed error. The Unity Asset Store version isn't supposed to require a subscription or account at Alteruna.com.

Alteruna:Error:license query failed!
Status: ConnectionError Request timeout Type: Light
Alteruna.Trinity.Exceptions.LicenseException: license query failed!
Status: ConnectionError Request timeout Type: Light
UnityEngine.Debug:LogError (object)
Alteruna.UnityLog:Present (Alteruna.Trinity.LogBase/Severity,string)
Alteruna.Trinity.LogBase:Log (Alteruna.Trinity.LogBase/Severity,string)
Alteruna.Trinity.LogBase:Error (string)
y/d/a:f ()
System.Threading.Tasks.TaskCompletionSource`1<UnityEngine.Networking.UnityWebRequest/Result>:TrySetResult (UnityEngine.Networking.UnityWebRequest/Result)
Alteruna.UnityWebRequestExtension/a:c (UnityEngine.AsyncOperation)
UnityEngine.AsyncOperation:InvokeCompletionEvent ()

**Comment by SimonAlteruna** [comment]
No, you don't need a subscription. But even free licenses from the asset store creates a license for you to manage your server.

Ensure you have a stable ethernet connection and do not block connections to Alteruna in any firewall settings.

Note that schools and other institutes can have a whitelist policy for what websites and services are allowed.

For fast quality support join our discord.
